packages/stylelint-plugin-mozaic/lib/splitSelector.js

```diff
@@ -0,0 +1,88 @@
+const _ = require('lodash')
+
+// eg .cm-foo-bar__baz--foo div
+// should return
+/* [
+    {
+        type: class,
+        subType: component,
+        string: 'cm-foo-bar__baz--foo',
+        bemStructure: [
+            { namespace: 'cm-' },
+            { block: 'foo-bar' },
+            { element: 'baz' },
```

Comment:
The element can also be in kebab-case, I don't know if you took this case into account?

Reply:
You're right! Every part of the structure should be in kebab-case! 👍